modules/govuk/manifests/apps/kibana.pp

```diff
@@ -1,13 +1,63 @@
 # == Class: govuk::apps::kibana
 #
-# Set up the Kibana application
+# Set up the Kibana application behind Signon
+#
+# === Parameters
+#
+# [*elasticsearch_host*]
+#   Which logs-elasticsearch host Kibana should connect to, must
+#   be a single host: https://github.com/elasticsearch/kibana/issues/214
 #
 # [*port*]
-#  Port that Kibana listens on
+#   Port that the Kibana wrapper listens on
+#
+# [*secret_key*]
+#   A secret key for Rack's session cookies
+#
+# [*signon_client_id*]
+#   A client ID for GOV.UK's Signon
 #
-class govuk::apps::kibana($port = '3202') {
-  govuk::app { 'kibana':
+# [*signon_client_secret*]
+#   The secret which matches signon_client_id
+#
+# [*signon_root*]
+#   The HTTP location of Signon
+#
+class govuk::apps::kibana (
+  $elasticsearch_host = 'logs-elasticsearch-1.management',
+  $port = '3202',
+  $secret_key = undef,
+  $signon_client_id = undef,
+  $signon_client_secret = undef,
+  $signon_root = undef,
+) {
+  $app_name = 'kibana'
+
+  govuk::app { $app_name:
     app_type => 'rack',
     port     => $port,
   }
+
+  Govuk::App::Envvar {
+    app => $app_name,
```

Comment:
Is there a reason this is a variable when it's not possible to change it when calling the class?